test/fixtures.py

```diff
@@ -5,7 +5,9 @@
 from io import StringIO
 from unittest.mock import MagicMock
 import pytest
```

Comment:
```suggestion

import pytest
```

We accustom to separate standard library, third-party and local imports with blank lines.

Reply:
fixed. 

In general, are you open to introduce `isort` into the linting config for this? 

IMO this eases contributions / reviews with many contributors. 

And while we're on it, what about black? 

( in any case, both not in this PR)

Reply:
> In general, are you open to introduce isort into the linting config for this?

Yes, I am. Thanks for suggesting it.

> And while we're on it, what about black?

Sure, sounds good to me too.

> ( in any case, both not in this PR)

Agreed. I don't have time to do it tough, but I'll be happy to review and merge such contribution.